pypck/module.py

```diff
@@ -681,13 +683,14 @@ class GroupConnection(AbstractConnection):
 
     def __init__(
         self,
+        registry: TaskRegistry,
```

Comment:
See above... Why pass the registry as an additional parameter, when you could use `conn.registry`?